detection-rules/link_coinbase_low_rep_or_shortened.yml

```diff
@@ -0,0 +1,50 @@
+name: "Brand Impersonation: Coinbase with suspicious links"
+description: |
+  Detects messages impersonating Coinbase with low reputation or url shortened links.
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and sender.email.domain.root_domain != "coinbase.com"
+  
+  // more than 0 less than 5 links
+  and 0 < length(body.links) < 5 
+  
+  // none of the links are to coinbase.com
+  and all(body.links, .href_url.domain.root_domain != "coinbase.com")
+  
+  // low rep or url shortened links found
+  and any(body.links, 
+      .href_url.domain.domain in $url_shorteners
+      
+      // exempting legitimate Google Maps shortener
+      and not strings.ilike(.href_url.url, "http?://goo.gl/maps*")
```

Comment:
yep! confirmed. it's not caught by the editor because of the extra newline after `or` before the parentheses. pulling that up:
![image](https://github.com/sublime-security/sublime-rules/assets/31489089/e35d6584-82b4-4cdc-b261-d682b6eb5ea3)